README.md

```diff
@@ -38,7 +38,15 @@ RGeo::Shapefile::Reader.open('myshpfil.shp') do |file|
   puts "First record geometry was: #{record.geometry.as_text}"
 end
 ```
+### Skipping validity checks
 
+If you have shapefiles that are failing [rgeo validity checks](https://github.com/rgeo/rgeo/blob/main/doc/Geometry-Validity.md) you can skip validity checks by passing `unsafe_methods: true` to the `RGeo::Shapefile::Reader#open` method:
+
+```ruby
+RGeo::Shapefile::Reader.open('myshpfil.shp', unsafe_methods: true) do |file|
```

Comment:
A late though, but why precising `unsafe_methods`? This is telling more about the code design than the current user intention. I'd go with `unsafe` solely (or `allow_unsafe`, which implies allow unsafe geometry). This is definitely not of the uttermost importance :)

Reply:
Good call! I like this naming better!